thunx-model/src/main/java/com/contentgrid/thunx/predicates/model/Scalar.java

```diff
@@ -30,6 +31,10 @@ static BooleanValue of(boolean value) {
         return new BooleanValue(value);
     }
 
+    static CollectionValue of(Collection<Scalar<?>> value) {
```

Comment:
Since a collection is no longer a scalar, this static method should be moved to `CollectionValue` (if you want to keep it around at all)